Headlines/src/Controllers/NewsTableViewController.swift

```diff
@@ -14,7 +14,8 @@ import ViewAnimator
 class NewsTableViewController: UITableViewController,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/ba933f4049a44fb3a8fc655051f4e975dcf6eb8e/Headlines/src/Controllers/NewsTableViewController.swift#L14"></span>Type body should span 200 lines or less excluding comments and whitespace: currently spans 324 lines</td>
    </tr>
  </tbody>
</table>